ui/store/actions.test.js

```diff
@@ -2074,7 +2078,7 @@ describe('Actions', () => {
                     },
                   },
                   options: {},
-                  methods: [...Object.values(EthMethod)],
+                  methods: ETH_4337_METHODS,
```

Comment:
Shouldn't this be `ETH_EOA_METHODS`? Or if we need also 4337 methods, we might need concat EOA + 4337?

Reply:
Yes this one should be EOA